### DISCUSSION TITLE
Commit signature verification - No user is associated with the committer email

### DISCUSSION THREAD

**Discussion by xormatrix** [community/community#109761] [Code Security]
### Select Topic Area

Question

### Body

As written in the title, When viewing signed commits I've pushed to GitHub, the following can be seen to the left of each commit's id: (Unverified): "No user is associated with the committer email". I have not gotten signed commits to work thus far on GitHub. I can see that my commits are being signed if I do: `git log --show-signature`, so the problem is with GitHub.

I've double-checked the email address in both my commits and in my GitHub settings, and the email address in my commits matches up to one in my GitHub settings (also checked that it's verified). I've tried using both GPG and SSH keys with two different email addresses, but to no avail. I tried with both the GPG master key and subkey, but as you may expect because of this post, it had no effect (honestly, I didn't expect it to either).

What could the problem be? Is there perhaps anything I've missed doing? I've checked out https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification and all relevant, related articles.

**Comment by xormatrix** [comment]
Forgot to provide an answer here but it turned out that my account was flagged for some reason, I discovered this only when I checked that my profile page gave a 404 not found error. What's worse is that I had no idea about it! If anyone has the same problem: commits cannot be verified if you have a flagged account. I created a support ticket on Feb 26 and received an answer in ~10 days luckily so it wasn't that bad in my case.

**Comment by ASCCON** [comment]
I am seeing the same problem but my account is not (obviously?) flagged.

**Reply by xormatrix** [reply]
Sorry for the late response. Did you fix this problem? As I wrote I created a support ticket and received help in 10 days or so. In case you're still having the issue, you can try creating a support ticket at https://support.github.com/tickets to get help from GitHub support.
